packages/playground/package.json

```diff
@@ -3,24 +3,43 @@
   "version": "0.0.54",
   "private": true,
   "scripts": {
+    "build": "just-scripts build",
     "start": "react-native start",
     "lint:fix": "just-scripts lint:fix",
     "lint": "just-scripts lint",
     "windows": "react-native run-windows"
   },
   "dependencies": {
+    "chalk": "^4.0.0",
     "react": "16.9.0",
     "react-native": "0.62.2",
     "react-native-windows": "0.0.0-master.46"
   },
   "devDependencies": {
     "@babel/core": "^7.8.4",
     "@babel/runtime": "^7.8.4",
+    "@types/jasmine": "2.8.7",
+    "@types/jquery": "^3.3.35",
+    "@types/node": "^10.14.8",
     "@types/react": "16.9.0",
     "@types/react-native": "^0.62.2",
+    "@wdio/appium-service": "5.12.1",
```

Comment:
I know this is still a draft, but this struck me as a bit unexpected. Why do we need to couple Playground to WinAppDriver, etc? Can we just load the RNTester bundle within E2ETest?